### PR TITLE
Feature/configure logging interceptor

### PR DESCRIPTION
**Pull request by clewis1**
Added logging interceptor and required interceptor config.

### CODE REVIEW

**Comment by ch-code-analysis** [comment]
CI: Security warnings found!

**Review by hepsimo — CHANGES_REQUESTED** [review]
Requesting changes to InterceptorConfig for consistency.

**Comment by ch-code-analysis** [comment]
CI: Security warnings found!

File: src/test/java/uk/gov/companieshouse/pscverificationapi/config/InterceptorConfigTest.java [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    @BeforeEach
+    void setUp() {
+        testConfig = new InterceptorConfig();
+    }
+
+    @Test
+    void addInterceptors() {
+        testConfig.addInterceptors(interceptorRegistry);
+
+        verify(interceptorRegistry.addInterceptor(any(RequestLoggingInterceptor.class))).order(2);
```
Comment (hepsimo) [inline src/test/java/uk/gov/companieshouse/pscverificationapi/config/InterceptorConfigTest.java:32]: Could add extra verify step:

```suggestion
        verify(interceptorRegistry.addInterceptor(any(TransactionInterceptor.class))).order(1);
        verify(interceptorRegistry.addInterceptor(any(RequestLoggingInterceptor.class))).order(2);
```

**Review by hepsimo — APPROVED** [review]
Changes look good, one extra verify suggested for the test

File: src/test/java/uk/gov/companieshouse/pscverificationapi/config/InterceptorConfigTest.java (hunk 1 above)
Comment (clewis1) [reply src/test/java/uk/gov/companieshouse/pscverificationapi/config/InterceptorConfigTest.java:32]: Added extra step.
I'll alter the addInterceptor test during the next interceptor story but will push this one through for now.

**Pull request merged**